components/02-elements/form/file-upload/file-upload.hbs

```diff
@@ -0,0 +1,24 @@
+<div class="file-upload {{ class }}">
+    <label class="file-upload__label" for="{{ name }}">
+        {{ labelText }}
+    </label>
+
+    <div class="file-upload__handler">
+        <input id="{{fileId}}"
+               class="file-upload__field"
+               title="title"
+               type="{{ type }}"
+               name="{{ name }}"
+               accept=".pdf,.doc,.png"
+               data-size="1024"
+        />
+
+        {{ render '@button' }}
```

Comment:
it's in elements because we have here other form elements and I just want to keep it in one folder. If you think it's a bad idea I can move it to the modules easily :)